Allow the use of jsx annotations (#702)
gajus/eslint-plugin-jsdoc: .README/rules/check-tag-names.md

}
```

#### `jsxTags`

> If this is set to `true`, tags used to control JSX output are allowed as all. This includes the the following tags:

Reviewer: ```suggestion
If this is set to `true`, all of the following tags used to control JSX output are allowed:
```

```
jsx
jsxFrag
jsxImportSource
jsxRuntime
```

> For more information, see the [babel documentation](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx)

Reviewer: ```suggestion
For more information, see the [babel documentation](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx).
```
